vipps-checkout-api.md

```diff
@@ -305,6 +305,17 @@ Vipps Checkout will expose a polling endpoint as described in our [Swagger](http
 
 In order to determine the status of a payment, you must poll epayment as described [here](https://vippsas.github.io/vipps-epayment-api/index.html#tag/QueryPayments/operation/getPayment).
 
+## Step 3a: If a transaction is authorized, capture payment
+
+If the state of session is `AUTHORISED`, the amount of the transaction was successfully _reserved_. In order to actually move the money and complete the payment, the transaction must be _captured_. If you ship physical goods, capture is not to be done until the shipment is sent to the customer. Otherwise, capture can happen anytime after the reservation happened as long as the customer has received what has been paid for. E.g. if you sell digital concert tickets and send them by email directly after a purchase.
```

Comment:
Maybe use "...until the shipment is _dispatched_ to the customer"?

Reply:
Rewrote it